src/snowcli/cli/main/outside_typer_global_context.py

```diff
@@ -0,0 +1,6 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class OutsideTyperGlobalContext:
+    debug_logs_and_tracebacks: bool
```

Comment:
Do we need this class? Maybe just pass value in SnowCliMainTyper constructor?

Reply:
It's a wrapper for values which need to be accessible outside of Typer but which can be modified after Typer instance start (for example after reading `--debug` flag).

I'm not sure how I would be able to modify the value (after reading `--debug` flag) without using a wrapper.

Reply:
I think you can use logic like this:
```
context = click.get_current_context()
debug = context.find_root().params.get("debug")
```
You can check if the flag was present or not, without need for additional state. WDYT? That's what we do for format flag

Reply:
I tried it before sending changes to review and it was throwing an exception - probably because being outside typer command execution.